src/Step/Deploy/SetCryptKey.php

```diff
@@ -73,6 +73,7 @@ public function execute()
         $key = $this->environment->getCryptKey();
 
         if (empty($key)) {
+            $this->logger->error("Crypt key missing.");
```

Comment:
I would add some longer message, explaining what's wrong + recommendations to fix it.

Reply:
Please use single-quotes.
Please do not use dot in the message end